src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/SqlSettings.java

```diff
@@ -68,6 +73,13 @@ public SqlSettings() {
         settings.put(METRICS_ROLLING_INTERVAL, Setting.longSetting(METRICS_ROLLING_INTERVAL, 60L, 1L,
                 NodeScope, Dynamic));
 
+        // Settings for cursor
+        settings.put(CURSOR_ENABLED, Setting.boolSetting(CURSOR_ENABLED, true, NodeScope, Dynamic));
```

Comment:
Have you updated the doc in PluginSettingIT?

Reply:
Will do.

Reply:
Done.